docs/courses/design/README.md

```diff
@@ -404,6 +404,7 @@ by writing a high-level course overview that consists of:
 - [a one-paragraph description](/courses/design/overview-description.md) (i.e., a sales pitch to students)
 - [half a dozen learning objectives](/courses/design/overview-objectives.md)
 - [a summary of prerequisites](/courses/design/overview-prerequisites.md)
+  and a list of any packages or other special technology the course will require.
```

Comment:
Need to clarify how this is different from the brainstorming. That is, make it clear that this is the finalized list of packages that need to be included.